packages/apiserver/src/scheduler/scheduler.service.ts

```diff
@@ -0,0 +1,13 @@
+import { Injectable } from '@nestjs/common';
+import { Cron } from '@nestjs/schedule';
+import { ReminderService } from '../reminder';
+
+@Injectable()
+export class SchedulerService {
+  constructor(private reminderService: ReminderService) {}
+
+  @Cron('0 * * * *')
```

Comment:
How often this method will be invoked acc to `0 * * * *` ? Each hour?

Reply:
yes